src/components/pages/home.tsx

```diff
@@ -19,13 +20,17 @@ const Home = () => {
 
   const value = (v: string) => (v === 'none' ? null : v);
 
-  const nextPrayerToday = value(todayPrayerTimes.nextPrayer()) as keyof adhan.PrayerTimes;
-  const nextPrayerTomorrow = value(tomorrowPrayerTimes.nextPrayer()) as keyof adhan.PrayerTimes;
+  const nextPrayerToday = value(todayPrayerTimes.nextPrayer()) as keyof adhan.PrayerTimes | null;
+  let nextPrayerTomorrow = null;
 
-  const nextPrayer = nextPrayerToday || nextPrayerTomorrow;
-  const nextPrayerTime = (
-    nextPrayerToday ? todayPrayerTimes[nextPrayerToday] : tomorrowPrayerTimes[nextPrayerTomorrow]
-  ) as Date;
+  let nextPrayer = nextPrayerToday;
+  let nextPrayerTime: FixType = nextPrayerToday ? todayPrayerTimes[nextPrayerToday] : null;
```

Comment:
Use `keyof adhan.PrayerTimes | null`  instead of `FixType`.

Reply:
Already tested :)
```
Type 'Coordinates' is not assignable to type 'keyof PrayerTimes | null'.ts(2322)
```

I "any" it for now because no time for it, otherwise i'll take a close look at it asap.